src/.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
```

Comment:
- [ ] [OPTIONAL] This linters file is meant to be added to the root of the project. Please kindly move it there because it is meant to apply to all files in the project